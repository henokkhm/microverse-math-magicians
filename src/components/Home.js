function Home() {
  return (
    <section>
      <h2>Welcome to our page</h2>
      <p>
        Lorem ipsum dolor sit amet, consectetur adipisicing elit. Commodi, ullam
        atque! Ad dicta velit et repudiandae eligendi sint maiores minus aliquid
        porro perspiciatis magnam animi assumenda illum neque eos possimus vitae
        similique maxime, nostrum natus. Eaque minima voluptas voluptatum
        possimus necessitatibus laborum quibusdam voluptatibus maiores ipsum
        deleniti sit ratione non dolore, natus reprehenderit voluptates rem
        molestias quidem ab enim eos officia atque soluta assumenda.
      </p>
      <p>
        Vel, aut error cumque veritatis esse quos facilis suscipit neque in!
        Cumque modi doloribus repellat ipsa, totam odio adipisci error
        doloremque nihil esse quos maiores vel quo a est libero deleniti quod
        omnis culpa labore quam? Tempore quaerat distinctio repudiandae
        reprehenderit corporis exercitationem ratione sit, aliquam excepturi
        nostrum laborum a ex optio inventore eum unde nulla, molestias rerum est
        non debitis praesentium soluta aspernatur. Asperiores, laborum.
      </p>
    </section>
  );
}

export default Home;
