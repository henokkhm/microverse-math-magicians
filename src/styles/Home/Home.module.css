.homePage {
  padding: 1rem;
  margin: 0 auto;
}

.pageTitle {
  position: relative;
  color: #3c1e1a;
  margin-bottom: 2rem;
}

.pageTitle::before {
  content: '';
  position: absolute;
  background: #f68644;
  border-radius: 1rem;
  bottom: -0.3rem;
  left: 0;
  width: 3rem;
  height: 0.25rem;
  z-index: -10;
}

.pageTitle::after {
  content: '';
  position: absolute;
  background: #fdbf9c;
  border-radius: 1rem;
  bottom: -0.3rem;
  left: 0;
  width: 5rem;
  height: 0.25rem;
  z-index: -20;
}

.paragraph {
  max-width: 50rem;
  padding: 1rem 0;
  color: #6e655a;
  font-size: 0.9rem;
}
