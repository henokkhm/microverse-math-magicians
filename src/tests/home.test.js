import renderer from 'react-test-renderer';
import Home from '../components/Home';

it('The homepage renders correctly', () => {
  const tree = renderer.create(<Home />).toJSON();
  expect(tree).toMatchSnapshot();
});
