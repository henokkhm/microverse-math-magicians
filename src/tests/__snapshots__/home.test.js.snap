// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<section
  className="homePage"
  data-testid="homepage-content"
>
  <h2
    className="pageTitle"
  >
    Welcome to our page!
  </h2>
  <p
    className="paragraph"
  >
    Lorem ipsum dolor sit amet, consectetur adipisicing elit. Commodi, ullam atque! Ad dicta velit et repudiandae eligendi sint maiores minus aliquid porro perspiciatis magnam animi assumenda illum neque eos possimus vitae similique maxime, nostrum natus. Eaque minima voluptas voluptatum possimus necessitatibus laborum quibusdam voluptatibus maiores ipsum deleniti sit ratione non dolore, natus reprehenderit voluptates rem molestias quidem ab enim eos officia atque soluta assumenda.
  </p>
  <p
    className="paragraph"
  >
    Vel, aut error cumque veritatis esse quos facilis suscipit neque in! Cumque modi doloribus repellat ipsa, totam odio adipisci error doloremque nihil esse quos maiores vel quo a est libero deleniti quod omnis culpa labore quam? Tempore quaerat distinctio repudiandae reprehenderit corporis exercitationem ratione sit, aliquam excepturi nostrum laborum a ex optio inventore eum unde nulla, molestias rerum est non debitis praesentium soluta aspernatur. Asperiores, laborum.
  </p>
</section>
`;
